src/renderer/components/+cluster-settings/components/cluster-kubeconfig.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Cluster } from "../../../../main/cluster";
+import { observer } from "mobx-react";
+import { SubTitle } from "../../layout/sub-title";
+import { autobind } from "../../../../common/utils";
+import { shell } from "electron";
+
+interface Props {
+  cluster: Cluster;
+}
+
+@observer
+export class ClusterKubeconfig extends React.Component<Props> {
+
+  @autobind()
+  openKubeconfig() {
+    const { cluster } = this.props;
+
+    shell.showItemInFolder(cluster.kubeConfigPath);
+  }
+
+  render() {
+    return (
+      <>
+        <SubTitle title="Kubeconfig" />
+
+        <span>
+          <a className="link value" onClick={this.openKubeconfig}>{this.props.cluster.kubeConfigPath}</a>
```

Comment:
Seems a bit odd that clicking on a link brings up a file select dialog. So user can view it?

Reply:
It has been link also in the past. Link allows user to find/edit kubeconfig easily.

![image](https://user-images.githubusercontent.com/1446224/114813809-86be5d80-9dbb-11eb-849b-b7f8858ed74a.png)